README.md

```diff
@@ -97,3 +97,16 @@ patrons_page = api_client.fetch_page_of_pledges(campaign_id, 10)
 next_cursor = api_client.extract_cursor(patrons_page)
 second_patrons_page = api_client.fetch_page_of_pledges(campaign_id, 10, cursor=next_cursor)
 ```
+
+
+Developing
+---
+1. Clone this repo
+2. `virtualenv venv`
+3. `source venv/bin/activate`
+4. `pip install -e .`
+5. `git checkout -b my-branch-name`
+6. make your edits, writing tests for any new functionality
+7. make sure tests pass with `???`
```

Comment:
TODO: @monokrome how were you running the tests you wrote?

Reply:
@21echoes Try `python setup.py test` - that should set up all of the dependencies for you!

Also, I would recommend using `python setup.py develop` instead of `pip install -e .` since it is the convention with setuptools and will symlink the module into your environment 😺

Reply:
🤔 i'm definitely under-informed here, but most of my google perusal said `pip install -e .` is the same as `python setup.py develop`, but better (so long as you have `pip` on your system). I'm down to use simpler things that make less assumptions tho!